tree/dataframe/test/datasource_ntuple.cxx

```diff
@@ -64,12 +64,23 @@ TEST_F(RNTupleDSTest, CardinalityColumn)
 {
    auto df = ROOT::Experimental::MakeNTupleDataFrame(fNtplName, fFileName);
 
-   // Check that the special column #<collection> works with jitting
-   auto max_njets = df.Define("njets", "__rdf_sizeof_jets").Max("njets");
-   EXPECT_EQ(2, *max_njets);
+   // Check that the special column #<collection> works without jitting...
+   // FIXME the type of the size columns should be std::size_t
```

Comment:
I'm not sure if that's correct.  What if we stored a >4GB vector and want to read it on a 32bit machine.  I think the type should be `std::uint64_t`.

Reply:
As per our discussion, we'll use `std::size_t` as the idiomatic "size of a thing" type in C++, leveraging RNTuple's type casting support introduced in https://github.com/root-project/root/pull/8067 (which is now a pre-requisite to this PR I guess)